nuxeo-features/nuxeo-platform-csv-export/src/main/java/org/nuxeo/ecm/platform/csv/export/computation/CSVProjectionComputation.java

```diff
@@ -125,4 +130,32 @@ protected String getData(String csv, String recordSeparator) {
         return csv.substring(csv.indexOf(recordSeparator) + recordSeparator.length());
     }
 
+    protected List<String> getList(Map<String, Serializable> params, String key) {
+        Serializable value = params.get(key);
+        List<String> values = new ArrayList<>();
+        if (value instanceof List<?>) {
+            List<?> objects = (List<?>) value;
+            for (Object object : objects) {
+                if (object != null) {
+                    values.add(object.toString());
+                }
+            }
+        } else if (value instanceof String[]) {
+            String[] array = (String[]) value;
+            values.addAll(Arrays.asList(array).stream().filter(Objects::nonNull).collect(Collectors.toList()));
+        } else if (value instanceof String) {
```

Comment:
We don't need the `String[]` or `String` cases. We should be strict in what we accept as a format. Maybe just `log.debug` the case where we get something unexpected

Reply:
Removed